addons/crm/views/crm_lead_views.xml

```diff
@@ -1298,4 +1300,12 @@
             }</field>
             <field name="search_view_id" ref="crm.view_crm_case_leads_filter"/>
         </record>
+
+        <record id="crm_lead_action_configure_properties_field" model="ir.actions.client">
```

Comment:
Maybe to put into a ir_actions_client_data.xml ? that way we keep :socks: sorted out